### ISSUE TITLE
Selecting Models in One Variable > Compare Models

### ISSUE THREAD

**Issue by lilyclements** [IDEMSInternational/R-Instat#1882]
For 'Model > One Variable > Compare Models' (and this issue will potentially apply for 'Model > x Variables') there is an issue in the objects you can select.

In 'Model > One Variable > Fit' we fit a model.
These models are then in the selector for 'Model > One Variable > Compare'
We should only be able to compare models from the same variable in 'Model > One Variable > Compare' and so need a way to monitor that once a fitted model is selected for the receiver, that any other models that you can select are now from the same variable.

One way this might work is if we create a new attribute to store the column name and dataset. Then we can edit selectors and receivers so they can choose models from the same variable.

**Comment by dannyparsons** [comment]
This is still needed. Should be part of discussions on models? @lilyclements

**Comment by lilyclements** [comment]
Yes, I mentioned this to David earlier today. This is part of discussions, along with that it shouldn't bring up Two-Variable, Three-Variable, etc, models in this dialog.
